src/app/modules/admin/components/profile-content/profile-content.component.html

```diff
@@ -0,0 +1,117 @@
+<div class="profile-content">
+    <h3 class="profile-content__title">Profile</h3>
+    <div class="profile-content__form">
+        <div class="profile-content__form-wrap">
+            <form [formGroup]="form" (ngSubmit)="updateProfile()">
+                <div class="form-group">
+                    <label for="name" class="form-group__label">Name</label>
+                    <input
+                        type="text"
+                        name="name"
+                        formControlName="name"
+                        #name
+                        id="name"
+                        class="form-control"
+                        required
+                    />
+                    <div
+                        *ngIf="form.get('name').touched && form.get('name').invalid"
+                        class="form-group__validator"
+                    >
+                        <small *ngIf="form.get('name').errors.required">
+                            Enter name
+                        </small>
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label for="email" class="form-group__label">Email</label>
+                    <input
+                        type="email"
+                        name="email"
+                        formControlName="email"
+                        #email
+                        id="email"
+                        class="form-control"
+                        required
+                    />
+                    <div
+                        *ngIf="
+                            form.get('email').touched &&
+                            form.get('email').invalid
+                        "
+                        class="form-group__validator"
+                    >
+                        <small *ngIf="form.get('email').errors.required">
+                            Enter email
+                        </small>
+                        <small *ngIf="form.get('email').errors.email">
+                            Enter valid email
+                        </small>
+                    </div>
+                </div>
+                <button
+                    class="form-btn"
+                    type="submit"
+                    [disabled]="form.invalid"
+                >
+                    Update profile
+                </button>
+            </form>
+        </div>
+        <div class="profile-content__form-logo">
+            <div class="change-logo">
+                <h4 class="title">Profile picture</h4>
+                <img
+                    src="{{ avatar$ | async }}"
+                    class="change-logo-img"
+                    alt="change-logo"
+                    *ngIf="!!profileAvatar"
+                />
+                <img
```

Comment:
as i understood it's default profile image, so name image as default-profile-image